python/hsfs/feature_group.py

```diff
@@ -3272,6 +3289,7 @@ def __init__(
         href=None,
         online_topic_name=None,
         topic_name=None,
+        notification_topic_name=None,
```

Comment:
I don't think spine feature groups should have the notification topic.
They are only used in the join operations.